oap-server/oal-rt/src/main/java/org/apache/skywalking/oal/rt/parser/SourceColumn.java

```diff
@@ -34,8 +34,10 @@ public class SourceColumn {
     private int length;
     private String fieldSetter;
     private String fieldGetter;
+    private final boolean groupByColInTopN;
 
-    public SourceColumn(String fieldName, String columnName, Class<?> type, boolean isID, int length) {
+    public SourceColumn(String fieldName, String columnName, Class<?> type, boolean isID, int length,
+                        boolean groupByColInTopN) {
```

Comment:
Is this `Col` meaning `Column`? I think this is duplicated as this class is column. My proposal was `Cond` for `Condition`.